lib/refactor-config.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   , directoryFilter :  null         // not supported yet
   , fileFilter      :  '.js'        // the extension of the file to upgrade
   , dryrun          :  true         // true|false if true no changes will be written to upgraded files
+  , moveStrict      :  false        // true|false if true moves 'use strict;' statement to the top of the file
```

Comment:
Ok. I just did this because you mentioned here https://github.com/thlorenz/browserify-ftw/issues/14#issuecomment-92396405 that it should be off.

I agree that this is a sensible default.